### PR TITLE
fix: fox lp follow-up fixes

### PR DESCRIPTION
**Pull request by stackedq**
## Description

Follow up of #2333 
basically name and icon changes for LP opportunity

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

### CODE REVIEW

File: src/components/StakingVaults/Cells.tsx [hunk 1]
```diff
@@ -26,9 +28,14 @@ type AssetCellProps = {
   postFix?: string
   showTeaser?: boolean
   showAssetSymbol?: boolean
+  icons?: Array<string>
 }
 
 const buildRowTitle = (asset: Asset, postFix?: string, showAssetSymbol?: boolean): string => {
+  if (asset.assetId === foxEthLpAssetId) {
```
Comment (0xdef1cafe) [inline src/components/StakingVaults/Cells.tsx:35]: can we please do this in a more extensible way with a map, we're going to have this problem in future for extra assets.

File: src/pages/Defi/components/OpportunityCard.tsx [hunk 2]
```diff
@@ -88,26 +90,44 @@ export const OpportunityCard = ({
 
   if (!asset) return null
 
+  const getOpportunityName = () => {
+    if (assetId === foxEthLpAssetId) return 'ETH-FOX UNI V2'
```
Comment (0xdef1cafe) [inline src/pages/Defi/components/OpportunityCard.tsx:94]: we're using this magic string in a lot of places, can we pop in it constants plz?

similarly the same concept of a map here applies for `assetId`s with manual name overrides

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]
small fixes plz

File: src/features/defi/providers/fox-eth-lp/constants.ts [hunk 3]
```diff
@@ -3,3 +3,4 @@ export const UNISWAP_V2_WETH_FOX_POOL_ADDRESS = '0x470e8de2ebaef52014a47cb5e6af8
 export const foxAssetId = 'eip155:1/erc20:0xc770eefad204b5180df6a14ee197d99d808ee52d'
 export const foxEthLpAssetId = 'eip155:1/erc20:0x470e8de2ebaef52014a47cb5e6af86884947f08c'
 export const MAX_ALLOWANCE = '0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF'
+export const foxEthLpOpportunityName = 'ETH-FOX UNI V2'
```
Comment (gomesalexandre) [inline src/features/defi/providers/fox-eth-lp/constants.ts:6]: Can we use this in fox page as well? 🙏🏽

**Review by gomesalexandre — APPROVED** [review]
Conceptually happy with this, just found more cases of the icon being "wrong":

image
image
image

Approving this for now as it needs @shapeshift/product confirmation, but from a consistency point of view it would make sense to use the same icons tuple (FOX+ETH) logic instead of the Uniswap icon

**Review by 0xdef1cafe — APPROVED** [review]

**Pull request merged**